tools/generate-helpers/pg-table-bindings/list.go

```diff
@@ -43,6 +43,8 @@ func init() {
 		&storage.NodeComponentEdge{}:                            resources.Node,
 		&storage.PermissionSet{}:                                resources.Role,
 		&storage.Pod{}:                                          resources.Deployment,
+		&storage.Policy{}:                                       resources.Policy,
```

Comment:
Policy is not indirectly scoped. Policy SAC -> policy SAC. Policy entry should not be here.

Reply:
https://github.com/stackrox/stackrox/pull/2289